docbook/reference/en/en-US/modules/Migration_from_older_versions.xml

```diff
@@ -12,6 +12,41 @@
         <ulink url="http://docs.jboss.org/resteasy/docs/resteasy-upgrade-guide-en-US.pdf">http://docs.jboss.org/resteasy/docs/resteasy-upgrade-guide-en-US.pdf</ulink>
     </para>
 </sect1>
+<sect1>
+    <title>Migrating to 4.0.0</title>
+    <para>
+       Version 4.0.0 of Resteasy is the first release to support JAX-RS 2.1 (see
+        <ulink url="https://jcp.org/en/jsr/detail?id=370">https://jcp.org/en/jsr/detail?id=370</ulink>),
+        which is discussed in <xref linkend="JAX-RS_2.1_additions"/>. Changes not related to the new spec
+        include the following.

```

Comment:
I'm probably showing up as very picky here, sorry, but afaiu we're going to only list topics that affect migration here, which is not directly related to them being about the jsr370 or not, right? IOW, in the release other non jaxrs21 changes are coming, but they are not going to be listed here if they do not affect migration, right?

Reply:
On the contrary, you're exactly right. I've removed the whole opening paragraph that referenced JAX-RS 2.1.